scripts/debug_rental_load.py

```
@@ -0,0 +1,459 @@
+        poll_tasks = [
+            poll_until_running(client, r.rental_id) for r in rentals
+        ]
+        poll_results: list[RentalState] = await asyncio.gather(*poll_tasks)
+
+        running_rentals = [r for r in poll_results if r.is_running]
+        not_running_rentals = [r for r in poll_results if not r.is_running]
+
+        print(f"\n{len(running_rentals)}/{len(rentals)} rentals reached running status")
+
+        if not_running_rentals:
+            print(f"\nRentals NOT running (will still be stopped):")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)